src/notification/notification.stories.ts

```diff
@@ -1,12 +1,69 @@
 import { storiesOf, moduleMetadata } from "@storybook/angular";
-import { action } from "@storybook/addon-actions";
+import { action, actions } from "@storybook/addon-actions";
```

Comment:
I don't see these imports being used anywhere ... that said, if there are some events we could hook up to the storybook action addon that would be awesome 👍